unity-game/Assets/Scripts/ElevatorScripts/ManualElevator.cs

```diff
@@ -0,0 +1,226 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ManualElevator : MonoBehaviour {
```

Comment:
`ManualElevator` and `AutoElevator` are extremely similar. Can you create a base `MovingPlatform` class and then make those extend it? (e.g. `public class MovingPlatform : MonoBehaviour` and `public class Manual Elevator : MovingPlatform`)